src/cursor.jl

```diff
@@ -394,8 +402,8 @@ isVariadic(c::Union{CXCursor,CLCursor}) = clang_Cursor_isVariadic(c) != 0
 
 # clang_Cursor_isExternalSymbol
 # clang_Cursor_getCommentRange
-# clang_Cursor_getRawCommentText
-# clang_Cursor_getBriefCommentText
+getRawCommentText(c::Union{CXCursor, CLCursor}) = convert(String, clang_Cursor_getRawCommentText(c))
+getBriefCommentText(c::Union{CXCursor, CLCursor}) = convert(String, clang_Cursor_getBriefCommentText(c))
```

Comment:
So, it's better to implement these two functions naively as:

```
function getRawCommentText(c::Union{CXCursor, CLCursor})
    cxstr = clang_Cursor_getRawCommentText(c)
    ptr = clang_getCString(cxstr)
    ptr == C_NULL && return ""
    s = unsafe_string(ptr)
    clang_disposeString(cxstr)
    return s
end
```

or define and reuse a `_get_cstr` helper function for internal use.

Reply:
Thanks for pointing this out! I will fix it.